src/libasr/codegen/asr_to_cpp.cpp

```diff
@@ -764,7 +764,7 @@ Kokkos::View<T*> from_std_vector(const std::vector<T> &v)
             case (ASR::binopType::Mul) : { last_expr_precedence = 5; break; }
             case (ASR::binopType::Div) : { last_expr_precedence = 5; break; }
             case (ASR::binopType::Pow) : {
-                src = "std::pow(" + left + ", " + right + ")";
+                src = "pow(" + left + ", " + right + ")";
```

Comment:
I think you need `std::pow`, don't you? Just like `std::abs`.

Reply:
Well, actually both will compile, but I can make the changes.

Reply:
The `<cmath>` has `std::pow`. It seems it compiles without `std::` in front, but then it can also be the `pow` from `<math.h>` which we do not want, so it's safer to always use the `std` namespace.

Reply:
I see, thanks!